.github/workflows/release-nmtc.yml

```diff
@@ -0,0 +1,39 @@
+name: Release nmtc
+
+on:
+  push:
```

Comment:
```
on:
  workflow_dispatch
```

https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
We do not need any parameters, but if it fails to appear/run, we can add some random one.

Reply:
Awesome!!! Will do